src/Project.js

```diff
@@ -129,19 +176,9 @@ export default class Project {
       }
     }
 
-    // Find triggers which match conditions
-    let matchingTriggers = [];
-    for (let i = 0; i < this.spritesAndStage.length; i++) {
-      const sprite = this.spritesAndStage[i];
```

Comment:
Interesting quirk about the old code I noticed: `spritesAndStage` is actually a getter which returns a new array every time it's called 😬

Reply:
[Who](https://github.com/leopard-js/leopard/blame/1e4e89edbb708c2e12f472c47d1c17f552d36152/src/Project.js#L177) would ever write such a thing 🙄